documentation/synthetic_tokens/tutorials/prerequisites.md

```diff
@@ -9,7 +9,7 @@ The following steps require the `git` CLI to be installed. If you are on Windows
 Clone the UMA [repo](https://github.com/UMAprotocol/protocol). Start in the top-level directory in this repository, `protocol/`.
 
 1. Install the latest stable version of [Node.js](https://nodejs.org/) and ensure that `npm` is installed along with it.
```

Comment:
Should we add some additional prerequisites? Like installing yarn and lerna?

Reply:
I think we should change our docs to reflect that we're using `yarn` now not `npm`, but its a bit confusing to have both `npx` and `yarn` present.